localbuild/README.rst

```diff
@@ -2,4 +2,4 @@ localbuild is mainly a copy of the build instructions from
 https://github.com/conda-forge/mss-feedstock/tree/master/recipe
 
 You may want to copy this directory to a different directory on your system
-and alter it for your needs.
\ No newline at end of file
+and alter it for your needs.
```

Comment:
there is als a newline added

rst maybe rendered differently here?

Reply:
in pycharm there is a `\n` defined. 
![image](https://github.com/Open-MSS/MSS/assets/278728/cb33d6ba-fb61-4909-b19d-3196e6513785)
it looks like it does somehow render rst here or at least did not show the icon for missing of a newline